samples/net/syslog_net/Kconfig

```diff
@@ -12,4 +12,20 @@ config NET_SAMPLE_SEND_ITERATIONS
 	  Send sample data this many times before exiting. A value of
 	  zero means that the defaults in the application are used.
 
+
+config NET_SAMPLE_NET_SERVER_RUNTIME
```

Comment:
The latter NET seems uncessary, `NET_SAMPLE_SERVER_RUNTIME` should be enough here.